tutorials/build_systems/easybuild/eb_test.py

```diff
@@ -0,0 +1,25 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class EasybuildMixin(rfm.RegressionTest):
+    @rfm.run_before('run')
+    def prepare_run(self):
+        self.modules = self.build_system.generated_modules
+
+
+@rfm.simple_test
+class BZip2Check(EasybuildMixin):
+    def __init__(self):
+        self.descr = 'This demonstrates the EasyBuild build system.'
+        self.valid_systems = ['daint:gpu']
+        self.valid_prog_environs = ['gnu']
+        self.modules = ['daint-gpu',
+                        'EasyBuild-custom']
```

Comment:
These make the test Daint-specific which is not very good for a tutorial. I would remove them. If users want to run the test on Daint, they should either load these modules outside ReFrame or edit the test file.